ramls/traits/includeResources.raml

```diff
@@ -0,0 +1,8 @@
+queryParameters:
+  include:
+    displayName: Related Resources
+    type: string
+    description: Include related resource obects, each representing a partnering of this title with a package.  Any bogus value, like `include=deciduousTrees`, will be silently ignored.
```

Comment:
A really minor nitpick with the spelling of "objects"

Reply:
fixed